packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useReducer, useMemo, useRef } from 'react';
 
 import TextInput from '../TextInput';
-import Dropdown, { DropdownProps } from '../Dropdown/Dropdown/Dropdown';
+import Dropdown, { DropdownProps } from '../Dropdown/Dropdown';
```

Comment:
This can just be `import Dropdown, { DropdownProps } from '../Dropdown';` since it has an index.ts

Reply:
on my way!

Reply:
@mshaaban0 done. I just adjusted it for `<Dropdown />` and `<Modal />` for now. the rest of export for Props on a separate branch.